### PR TITLE
Git/week1

### PR DESCRIPTION
**Pull request by daxiousm**
I just added two favorite food recipes and countries that i visited

### CODE REVIEW

File: git/week1/countries.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@ spain
 UK
 USA
 France
-Seychelles
\ No newline at end of file
+Seychelles
+Italy
```
Comment (daxiousm) [inline git/week1/countries.txt:7]: new country added

File: git/week1/countries.txt [hunk 2]
```diff
@@ -3,4 +3,5 @@ spain
 UK
 USA
 France
-Seychelles
```
Comment (daxiousm) [inline git/week1/countries.txt]: i don't know why this got deleted and added back below, i. haven't done anything regarding this text.

Comment (martamatos) [reply git/week1/countries.txt]: That is a git thing, i also don't understand why it happens :/

**Comment by martamatos** [comment]
Perfect! Please go ahead and merge! :D

**Pull request merged**